Jenkinsfile

```diff
@@ -16,6 +16,7 @@ pipeline {
     ITS_PIPELINE = 'apm-integration-tests-selector-mbp/master'
     MAVEN_CONFIG = '-Dmaven.repo.local=.m2'
     OPBEANS_REPO = 'opbeans-java'
+    JAVA_VERSION = "${params.JAVA_VERSION}"
```

Comment:
This is required when using params for the very first build, since the params is not loaded correctly for the very first build.